### PR TITLE
Merging contributions from DJ, Noel and Shane to a final stable master branch sprint 5

### PR DESCRIPTION
**Pull request by dgiri12**
Bar graph that shows data analytics has been added as a checkbox that triggers it’s visibility.

Bulk Customer Order and Bulk supplier order successfully print their result, talk to their sql tables, and respond to simulation mode

Core CRUD function buttons talk to required sql tables, respond to simulation mode and use the same text area for all input output operations.

Code refactoring in main.java

### CODE REVIEW

**Review by NoelC9 — APPROVED** [review]
Went over changes with dj. Not a lot of code was changed. We just had to restructure files so that we wouldn't get any file conflicts. This version is ready for tomorrow.

File: src/main/java/com/cs3250p1/project1/BarChartEx.java [hunk 1]
```diff
@@ -1,5 +1,11 @@
 package com.cs3250p1.project1;
```
Comment (dgiri12) [inline src/main/java/com/cs3250p1/project1/BarChartEx.java:1]: author of BarChartEx.java is @NoelC9

File: src/main/java/com/cs3250p1/project1/GUI.java [hunk 2]
```diff
@@ -15,7 +15,6 @@
 import java.util.ArrayList;
```
Comment (dgiri12) [inline src/main/java/com/cs3250p1/project1/GUI.java:15]: author for gui.java is @dgiri12  and @sbowma15

File: src/main/java/com/cs3250p1/project1/Main.java [hunk 3]
```diff
@@ -8,22 +8,20 @@
 import java.util.List;
```
Comment (dgiri12) [inline src/main/java/com/cs3250p1/project1/Main.java:8]: author @dgiri12  and. optimized by @NoelC9

File: src/main/java/com/cs3250p1/project1/Main.java [hunk 4]
```diff
@@ -83,6 +94,13 @@ public void windowClosing(WindowEvent e) {
                 System.out.println("Simulator data cleared!");
```
Comment (dgiri12) [inline src/main/java/com/cs3250p1/project1/Main.java:94]: window listener made by @sbowma15

File: src/main/java/com/cs3250p1/project1/OrderDAO.java [hunk 5]
```diff
@@ -9,7 +9,7 @@
 import java.util.ArrayList;
```
Comment (dgiri12) [inline src/main/java/com/cs3250p1/project1/OrderDAO.java:9]: made by @NoelC9

**Review by layer8sudostate — APPROVED** [review]
Looks like the bin files were changed but I couldn't tell for sure since it can't display those files. Gradle seems to be updated since the files have changed. Looks like bar chart files have been added per the user story. I believe that all other files have been cleaned up for this sprint.

**Pull request merged**